narwhals/dataframe.py

```diff
@@ -195,17 +195,28 @@ class DataFrame(BaseFrame):
     Examples:
         Constructing a DataFrame from a dictionary:
 
+        >>> import pandas as pd
         >>> import polars as pl
         >>> import narwhals as nw
         >>> data = {"a": [1, 2], "b": [3, 4]}
         >>> df_pl = pl.DataFrame(data)
-        >>> df = nw.DataFrame(df_pl)
+        >>> df_pd = pd.DataFrame(data)
+        >>> df = nw.from_native(df_pl)
```

Comment:
No need to define a narwhals dataframe here thats what the function you created later will do and lets define the polars dataframe again so we can pass it into the function, something like this:
df_pl = pl.dataframe(data)

Reply:
Hello Matthew,  you may have commented on an already changed fix. I made corrections to this when Marco pointed it out to me. Could you please highlight the method by which this mistake was found? And the line number as well.

Reply:
hey - it doesn't looked like this has already changed. @mfonekpo perhaps you made the changes but didn't push them? they still show here

Reply:
Alright @MarcoGorelli. I will have a second look at it right away